extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -36,7 +35,6 @@ import {toggle} from '../../../src/style';
 installActivityService(AMP.win);
 installCidService(AMP.win);
 installCryptoService(AMP.win);
-installStorageService(AMP.win);
```

Comment:
browser cache expiration on non Google cache origins. they don't skew on Google cache (https://cdn.ampproject.org/c/s/www.mypage.com) because we keep all the versions inline but we can't do that for www.mypage.com